driver/gl/gl.go

```diff
@@ -174,7 +174,7 @@ func (c *glCanvas) newGlImageTexture(obj fyne.CanvasObject) uint32 {
 
 				icon, err := oksvg.ReadIconStream(file)
 				if err != nil {
-					fyne.LogError("SVG Load error:", err)
```

Comment:
Search and replace strikes again?

Reply:
yeah, i should have looked more carefully at what it said it was changing, the dangers of coding while holding a 1 year old.